### DISCUSSION TITLE
Нормальный мэр.

### DISCUSSION THREAD

**Discussion by sarkazm-marazm** [octothorp-team/public-dobrograd#792] [Предложения]
Предлагаю добавить наборного мэра. К примеру ситуация также относится к муниципалам, игрок набрал нужно количество часов и становиться мэром. Он же новичок и ничего не понимает, поэтому город будет страдать. Нету законов, может и есть но не нормальные. Выдача лицензий происходит без РП отыгровки. Обустройки мэрии скорей всего не будет. Но и к чему такой мэр? 
Лучше всего будет сделать единственную заявку на мэра. Человек подаёт по критериям, также как и в других любых организациях. Как по мне это будет отлично и оптимально.

**Comment by outclown** [comment]
Ждем систему организаций, тогда это будет реализтовано
